src/apint/utils.rs

```diff
@@ -209,6 +245,7 @@ impl ApInt {
     }
 
     /// Returns a slice over the `Digit`s of this `ApInt` in little-endian order.
+    #[inline(always)]    
```

Comment:
Please reconsider if using `#[inline(always)]` in so many cases is really the right option. I can think of cases where the inlining is poisonous to caller code. So I think in the general case it is better to let the compiler decide upon every use case.